src/js/index.js

```diff
@@ -1,5 +1,10 @@
 import '../css/index.css';
+import { addMessageToChatHistory } from './chatWindow';
+import { callbackWhenEnterIsPressed, callbackOnFormSubmit } from './eventHandlers';
 
 const helloWorld = () => console.log('Hello!');
 
+callbackWhenEnterIsPressed(addMessageToChatHistory);
```

Comment:
In the truest sense, these aren't callback functions. I think the real intent of these functions is to establish the event listeners.

Reply:
Yeah, I think that naming is a bit confusing. I renamed them to onEnterPress and onFormSubmit, Alice and I thought it was easier to read something like "onEnterPress(addMessageToChatHistory)".